apps/damap-frontend/src/app/components/layout/layout.component.css

```diff
@@ -366,3 +368,28 @@ svg {
   padding-left: 22px;
   font-weight: 400;
 }
+
+.flex-row-container {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
+
+.page-info-icon {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 18px;
+}
+
+.example-button-container button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.info-box-container {
+  font-size: small;
+  font-weight: 300;
+  color: var(--user-navbar-color) !important;
```

Comment:
`!important is  not needed in this case`. You could also think about removing the color property altogether, since the color that mat3 chooses by default is very similar to the one you used.